### DISCUSSION TITLE
Content Include `.feature` files in `.csproj`

### DISCUSSION THREAD

**Discussion by KonstantinosArch** [reqnroll/Reqnroll#57] [Q&A] [answered]
Hello all!
Why do we need to have this in `.csproj` file?
```xml
  <ItemGroup>
    <Content Include="Features\*.feature"/>
  </ItemGroup>
```
When using SpecFlow it wasn't there. And if I remove it, the IDE can not identify `.feature` files, only the `.feature.cs` files.

**Comment by gasparnagy** [comment]
You don't need it. Maybe Visual Studio added that by mistake. 

What do you mean by "IDE can not identify .feature files"? Could you please share a screenshot?

**Reply by ajeckmans** [reply]
I had the same issue. The feature files are not included in the project unless you add those lines to the csproj files. With the specflow package the feature files were automatically included.

I wasn't sure whether this was a regression in the new rider plugin or reqnroll, but if it also surfaces in visual studio then probably the latter.

**Reply by KonstantinosArch** [reply]
I am using Rider with the [latest build](https://github.com/reqnroll/Reqnroll.Rider/actions/runs/7951275864) of the Rider.Reqnroll plugin. 
Here is how it looks when having the above snippet in `.csproj` and how it should be looking even when remove it:
![image](https://github.com/reqnroll/Reqnroll/assets/23631145/b1bd04b1-37a2-4503-a581-9ca7f648beff)
and this is how it actually looks when removing it:
![image](https://github.com/reqnroll/Reqnroll/assets/23631145/d7d4f8b2-e153-4d76-9953-e7c95e8f3114)

**Reply by gasparnagy** [reply, marked as answer]
@konarx @ajeckmans I can confirm, that with Visual Studio the projects work fine without that include, so this seems to be an issue with the Rider plugin.
